### PR TITLE
Orbiter clockwork adaptor mod

### PR DESCRIPTION
**Pull request by spacelab2021**
* [X ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X ] Submitted STL files are oriented and scaled properly for printing.
  * [X ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
This mod allows the full integration of the Orbiter v1.5 extruder into the Afterburner system. It includes a chain anchor for 2.4 and SW, and Connector Cover. It also includes a filament quick release lever which is an improvement for the Orbiter since there isn't one currently available. 
#### How was it tested? 
It was tested on my Voron 2.4 that is almost ready for a serial. I also converted a ender 3 to a SW and tested it on that platform. It still needs more testing on other printers, but I believe it to be publicly ready for release and build.

#### Any background context you want to provide?
I wanted to use the Galileo, but I went with the Orbiter and didn't find any suitable clockwork adaptors that included the chain anchor, cover, and a filament quick release. I had to created it and I saw there was a need for the quick release in Discord as few people complained about it previous.

#### Screenshots (if appropriate)
Available in the mod folder. Lots of pictures and renders are available there.

### CODE REVIEW

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Please remove the `LICENSE` file from your directory, this whole repository is GPLv3 and there's a license file at the root level.

`[a]_Wire_Cover.stl` has an unsupported overhang at the bottom of the screw hole:
Screen Shot 2021-09-24 at 20 43 05

Also see changes in the README below.

File: printer_mods/README.md [hunk 1]
```diff
@@ -220,6 +220,7 @@ like so:
 | sse018 | [Octopus DIN Rail bracket](./sse018/Octopus_DIN_Rail_bracket) | A Din rail mount for BTT Octopus board | :x: | :x: |:heavy_check_mark: | :x: |
 | Slidr | [PanzerChain 2.4](./Slidr/PanzerChain2.4) | Energychain for V2.4 | :x: | :x: | :heavy_check_mark: |:x:|
 |sloscotty|[Switchwire Wall Plate](./sloscotty/SW_Wall_Plate)|SW plate for enclosure|:x:|:x:|:x:|:heavy_check_mark:|
+| spacelab2021 | [Orbiter Clockwork Mount](spacelab2021/Orbiter-Clockwork) | A clockwork adaptor for the Orbiter v1.5 Extruder with full integration into the Afterburner. Includes a filament quick release lever for simple filament loading and unloading. | :x: | :heavy_check_mark: | :heavy_check_mark: | :grey_question: |
```
[the remark this answers is not in the document]
Comment (spacelab2021) [reply printer_mods/README.md]: I have updated the readme and added the checkmarks for the compatible printers.

File: printer_mods/spacelab2021/Orbiter-Clockwork/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Orbiter Clockwork Module
+This Clockwork module allows the use of the Orbiter v1.5 Extruder in the Voron Afterburner.
+
+## Features
+One of the main issues I observed when using the orbiter was a lack of a quick filament release in additional to a lack of a suitable clockwork adapter that would swap an orbiter directly into the Afterburner ecosystem. Many of the issues observed were addressed with this Clockwork Module. This version provides a chain anchor for the Voron 2.4 and the Switchwire, a Cable Cover, and a quick filament release lever.
```
[the remark this answers is not in the document]
Comment (spacelab2021) [reply printer_mods/spacelab2021/Orbiter-Clockwork/README.md]: It is compatible with the Switchwire, so I updated the readme to show the checkmark.

File: printer_mods/spacelab2021/Orbiter-Clockwork/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* SHCS 3x8mm
+  * 1x for Cable Cover to secure to Cable Chain Anchor
+* SHCS 3x16mm
+  * 1x for Filament Release Lever
+* SHCS 3x20
+  * 2x for Motor Mount to Chain Anchor  
+
+  
+## Assembly Guide
+* [Orbiter Clockwork Assembly Guide](./docs/Orbiter_Cockwork_Manual.pdf)
```
[the remark this answers is not in the document]
Comment (spacelab2021) [reply printer_mods/spacelab2021/Orbiter-Clockwork/README.md]: I updated this spelling mistake in both the filename and the link.

File: printer_mods/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (spacelab2021) [reply printer_mods/README.md]: The license has also been removed from my mod folder.

[the remark this answers is not in the document]
Comment (spacelab2021) [reply printer_mods/README.md]: I modified the STL file to remove the overhang, so it should be fixed. I also updated the Step file to include the updated connector cover.

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Pull request merged**